### PR TITLE
SW-1827 Fix see all accessions results + related fixes

### PR DESCRIPTION
**Pull request by karthikbtf**
- Clicking on accessions by status in the dashboard works only once
- Subsequent clicks don't show the correct response because the previous filter isn't cleared
- Added a fix to clear filters if we have a new filter request
- Also added fix for SW-1833 (don't show notes icon if there are no notes)

### CODE REVIEW

File: src/components/seeds/summary/index.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export default function SeedSummary(props: SeedSummaryProps): JSX.Element {
                     <Typography fontSize='20px' color='#000000' paddingBottom='8px' paddingLeft='28px'>
                       {strings.ACCESSION_BY_STATUS}
                     </Typography>
-                    <Link className={classes.accessionsLink} to={APP_PATHS.ACCESSIONS}>
+                    <Link className={classes.accessionsLink} to={`${APP_PATHS.ACCESSIONS}?stage=}`}>
```
Comment (constanzauanini) [inline src/components/seeds/summary/index.tsx:192]: Why is this necessary?

File: src/components/seeds/database/index.tsx [hunk 2]
```diff
@@ -201,8 +201,9 @@ export default function Database(props: DatabaseProps): JSX.Element {
     const stage = query.get('stage');
     const facilityId = query.get('facilityId');
     let newSearchCriteria = searchCriteria || {};
-    if (stage) {
-      if (ACCESSION_STATES.indexOf(stage) !== -1) {
+    if (stage || query.has('stage')) {
```
Comment (constanzauanini) [inline src/components/seeds/database/index.tsx:204]: if stage is equal to query.get('stage'); why is this or necessary?

File: src/components/seeds/summary/index.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/seeds/summary/index.tsx:192]: We need a way to say load all stages (since there might be an existing filter). Using `stage=` for that. Thought about `stage=*` but `stage=` would clear previous filters, and `stage=<something` would set the next filter. Open to suggestions.

File: src/components/seeds/database/index.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:204]: query.get('stage') will capture `stage=<something>` query.has('stage') will capture if the key stage exists - even if there is no value. query.get('stage') will only capture value.

Comment (constanzauanini) [reply src/components/seeds/database/index.tsx:204]: Oh, ok, thanks for explaining that

File: src/components/seeds/summary/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/seeds/summary/index.tsx:192]: I think this is ok for now. I didn't realize this wasn't working more than 1 time

**Review by constanzauanini — APPROVED** [review]

File: src/components/seeds/summary/index.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/seeds/summary/index.tsx:192]: Thanks. Yes I didn’t realize that either. I’ll update the web-components version too and update this PR.
